src/Microsoft.ML.Transforms/PermutationFeatureImportanceExtensions.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Transforms;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+
+namespace Microsoft.ML
+{
+    public static class PermutationFeatureImportanceExtensions
+    {
+        public static ImmutableArray<RegressionEvaluator.Result>
+            PermutationFeatureImportance(
```

Comment:
>PermutationFeatureImportance [](start = 12, length = 28)

`CalculateFeatureImportance` (to make it a verb). 
Also add a summary comment and describe all the parameters